Build.ps1

```diff
@@ -20,7 +20,6 @@ $projects =
   "src\Microsoft.Azure.WebJobs.Host\WebJobs.Host.Sources.csproj",
   "src\Microsoft.Azure.WebJobs.Logging\WebJobs.Logging.csproj",
   "src\Microsoft.Azure.WebJobs.Logging.ApplicationInsights\WebJobs.Logging.ApplicationInsights.csproj",
-  "src\Microsoft.Azure.WebJobs.Extensions.Storage\WebJobs.Extensions.Storage.csproj",
```

Comment:
These changes thus far just remove all dependencies on this project right? Ultimately we'll want to delete this code completely to remove any possibility of confusion since the dev branch (which will have v5 version of Host.Storage) will depend only on Track2, and we'll have a separate maintenance branch for v4 Host.Storage and Host.Storage.Extensions. We'll be building/servicing the v4 versions from that other branch.